### PR TITLE
Consistency of progress indication, round to nearest integer

### PR DESCRIPTION
**Pull request by AllenBW**
fixes https://bugzilla.redhat.com/show_bug.cgi?id=1614824

### CODE REVIEW

**Comment by AllenBW** [comment]
@michaelkro Yeah!  Wax is always gonna be an integer, ensuring `now` is also one... would beeeeeee a fix... also, data manipulation in the reducer is far preferred over the component, thanks for that reminder <3

File: app/javascript/react/screens/App/Plan/PlanReducer.js [hunk 1]
```diff
@@ -147,7 +147,7 @@ const processVMTasks = vmTasks => {
 
       taskDetails.diskSpaceCompleted = percentComplete * taskDetails.totalDiskSpace;
       taskDetails.diskSpaceCompletedGb = numeral(taskDetails.diskSpaceCompleted).format('0.00b');
-      taskDetails.percentComplete = Math.round(percentComplete * 1000) / 10;
+      taskDetails.percentComplete = Math.round((percentComplete * 1000) / 10);
```
Comment (michaelkro) [inline app/javascript/react/screens/App/Plan/PlanReducer.js]: The ghost of my former life as a mathematician won't let me leave this be, lol

Can we change this to `Math.round(percentComplete * 100)` 🙏

Comment (AllenBW) [reply app/javascript/react/screens/App/Plan/PlanReducer.js]: This is reasonable, can totally find reason this friday, promise.

**Comment by fdupont-redhat** [comment]
@miq-bot remove_label gaprindashvili/yes
@miq-bot add_label hammer/yes

**Comment by simaishi** [comment]
Removed `hammer/yes` as this PR was merged before hammer branch was created

**Pull request merged**